templates/dockerhub/Dockerfile

```diff
@@ -28,9 +28,11 @@ LABEL version="<%= version %>" \
 VOLUME ["/actions"]
 WORKDIR /actions
 
-COPY --from=build /tmp/fakeroot/ /
+RUN mkdir -p /etc/ssl/certs
+COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
question: I don't understand this one: I read that we copy a file at the same place, does this make sens?

Reply:
Inside the base image, busybox:glibc there is not cert or packet manager. So we copy the certs from the build image with the `--from`